src/table/components/TableBodyWrapper.tsx

```diff
@@ -31,15 +31,22 @@ function TableBodyWrapper({
   const isSelectionsEnabled = !constraints.active && !constraints.select;
   const columnRenderers = useMemo(
     () =>
-      JSON.parse(columnsStylingIDsJSON).map((stylingIDs) => getCellRenderer(!!stylingIDs.length, isSelectionsEnabled)),
+      JSON.parse(columnsStylingIDsJSON).map((stylingIDs: string[]) =>
+        getCellRenderer(!!stylingIDs.length, isSelectionsEnabled)
+      ),
     [columnsStylingIDsJSON, isSelectionsEnabled]
   );
   const bodyCellStyle = useMemo(() => getBodyCellStyle(layout, theme), [layout, theme]);
   const hoverEffect = layout.components?.[0]?.content?.hoverEffect;
   const cellStyle = { color: bodyCellStyle.color, backgroundColor: theme.table.backgroundColor };
-
   useEffect(() => {
-    addSelectionListeners({ api: selectionsAPI, selectionDispatch, setShouldRefocus, keyboard, tableWrapperRef });
+    addSelectionListeners({
+      api: selectionsAPI as ExtendedSelectionAPI,
```

Comment:
@LiKang6688  set `api `as `selectionAPI ` is the reason of using ExtendedSelectionAPI here.